### ISSUE TITLE
ImportError: No module named numpy.distutils.core

### ISSUE THREAD

**Issue by LuisBL** [rainwoodman/sharedmem#9]
With the following pip, setuptools::

```
~/lab/sandbox/ble$ pip --version
pip 8.1.2 from ~/local/lib/python2.7/site-packages (python 2.7)
~/lab/sandbox/ble$ python --version
Python 2.7.10
~/lab/sandbox/ble$ python -c 'import setuptools as st; print st.__version__'
28.6.1
~/lab/sandbox/ble$
```

I get this errror::

```
~/lab/sandbox/ble$ pip install sharedmem
Collecting sharedmem
Using cached sharedmem-0.3.3.tar.gz
  Complete output from command python setup.py egg_info:
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-ufEIrD/sharedmem/setup.py", line 1, in <module>
      from numpy.distutils.core import setup, Extension
  ImportError: No module named numpy.distutils.core
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ufEIrD/sharedmem/
~/lab/sandbox/ble$
```

I've try to pip install lower version with no success

**Comment by rainwoodman** [comment]
Hi

It was fixed in #8. I will mark a new release.

Yu

**Comment by rainwoodman** [comment]
0.3.5 should have fixed the problem. Note that the anonymous memory map object is accessed like dense numpy arrays, thus numpy is still a dependency.
